frontend/packages/kubevirt-plugin/src/components/vms/menu-actions.tsx

```diff
@@ -85,6 +87,37 @@ const menuActionRestart = (
   };
 };
 
+// TODO Remove function before merging
```

Comment:
The pause option isn't part of the feature. I was having auth troubles with doing it via the API, so I added the option to pause to the actions/kebab menu so that I can test the unpause functions manually.